presets/wordpress.json

```diff
@@ -1,5 +1,8 @@
 {
     "preset": "jquery",
+    "requireDotNotation": {
+        "allExcept": ["snake_case", "keywords"]
```

Comment:
It is better to make such changes more visible i.e. have special commit and link on why this was done